static/app/views/releases/list/index.tsx

```diff
@@ -244,6 +247,53 @@ class ReleasesList extends AsyncView<Props, State> {
     );
   }
 
+  renderAlertBanner() {
+    const {selection, organization} = this.props;
+
+    const selectedProjectId =
+      selection.projects && selection.projects.length === 1 && selection.projects[0];
+    const selectedProject = organization.projects.find(
+      p => p.id === `${selectedProjectId}`
+    );
+
+    if (!selectedProject) {
+      return null;
+    }
+
+    return (
+      <Feature features={['organizations:release-adoption-chart']}>
+        <Projects orgId={organization.slug} slugs={[selectedProject.slug]}>
+          {({projects, initiallyLoaded, fetchError}) => {
+            const project = projects && projects.length === 1 && projects[0];
+            const projectHasReleases =
+              project &&
+              project.hasOwnProperty('features') &&
+              (project as Project).features.includes('releases');
+
+            if (!initiallyLoaded || fetchError || !project || projectHasReleases) {
+              return null;
+            }
+
+            return (
+              <Alert type="info" icon={<IconInfo size="md" />}>
```

Comment:
Let's be sure to check the platform of the selected project against release health supported platforms:
https://github.com/getsentry/sentry/blob/ff35b49749816c62fd720628c9cc619c82aad7c9/static/app/data/platformCategories.tsx#L164